Agents/HeatEmissionAgent/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
+
+services:
+  heat-agent:
+    image: heat-agent:2.0.0
```

Comment:
standardise the agent name as mentioned before

Reply:
Done